e2e/testnet_wallet.test.js

```diff
@@ -0,0 +1,155 @@
+  await device.launchApp({ newInstance: false });
+
+  await waitFor(element(by.text("Finished"))).toBeVisible().withTimeout(sync_timeout)
+
+  // waiting for starting the sync process again
+  // await waitFor(element(by.id('syncreport.currentbatch'))).toBeVisible().withTimeout(ui_timeout);
+
+  // getting current batch & total batches from the screen
+  const batches = element(by.id('syncreport.currentbatch'));
+  const batches_attributes = await batches.getAttributes();
+  const batchNum = Number(batches_attributes.text.split(':')[1].split('of')[0]);
+  const batchesNum = Number(batches_attributes.text.split(':')[2]);
+
+  // getting blocks now synced from the screen
+  const blockssyncednow = element(by.id('syncreport.syncednow'));
+  const blockssyncednow_attributes = await blockssyncednow.getAttributes();
+  const blockssyncednowNum = Number(blockssyncednow_attributes.text.split(' ')[0]);
+
+  // getting blocks not yet sync from the screen
+  const blocksnotyetsynced = element(by.id('syncreport.notyetsynced'));
+  const blocksnotyetsynced_attributes = await blocksnotyetsynced.getAttributes();
+  const blocksnotyetsyncedNum = Number(blocksnotyetsynced_attributes.text.split(' ')[0]);
+
+  // calculating total blocks in this sync process
+  const blockstotalNum = blockssyncednowNum + blocksnotyetsyncedNum;
+
+  // getting blocks per batch or batch size from the screen
+  const batchsize = element(by.id('syncreport.blocksperbatch'));
+  const batchsize_attributes = await batchsize.getAttributes();
+  const batchsizeNum = Number(batchsize_attributes.text);
+
+  log.info('batches', batchNum);
+  log.info('total batches', batchesNum);
+  log.info('blocks sync now', blockssyncednowNum);
+  log.info('blocks not yet sync', blocksnotyetsyncedNum);
+  log.info('blocks total', blockstotalNum);
+  log.info('batch size:', batchsizeNum);
+
+  // a couple of examples:
+  // batch: 1  -> means blocks between 0 and 100
+  // batch: 33 -> means blocks between 3200 and 3300
+  if (blockssyncednowNum < (batchNum * batchsizeNum) - batchsizeNum || blockssyncednowNum > (batchNum * batchsizeNum)) {
+    fail('The synced blocks do not align with the synced batches');
+  }
+  
+  if (blockstotalNum < (batchesNum * batchsizeNum) - batchsizeNum || blockstotalNum > (batchesNum * batchsizeNum)) {
+    fail('The total blocks in this process do not align with the total of batches');
+  }
+}
+
+async function send(address) {
+  // since we ended up in the sync report screen
+  await waitFor(element(by.text('CLOSE'))).toBeVisible().withTimeout(ui_timeout);
+  await element(by.text('CLOSE')).tap();
+  
+  await waitFor(element(by.id('header.drawmenu'))).toBeVisible().withTimeout(ui_timeout);
+  await element(by.id('header.drawmenu')).tap();
+  await waitFor(element(by.id('menu.settings'))).toBeVisible().withTimeout(ui_timeout);
+  await element(by.id('menu.settings')).tap();
+  await waitFor(element(by.id('settings.sendalls-enable'))).toBeVisible().withTimeout(ui_timeout);
```

Comment:
I think the `TestID` is correct, but you need to scroll a little bit first to find this field.